src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/SocketOutputProducer.cs

```diff
@@ -6,18 +6,14 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Server.Kestrel.Internal.Infrastructure;
-using Microsoft.AspNetCore.Server.Kestrel.Internal.Networking;
 using Microsoft.Extensions.Internal;
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Http
 {
-    public class SocketOutput : ISocketOutput
+    public class SocketOutputProducer : ISocketOutput
```

Comment:
Since we're no longer interacting directly with a socket, rename this to `OutputProducer`. Rename the interface too.